cron/alerts/main.go

```diff
@@ -224,10 +230,6 @@ func makeSESMessage(charSet, subject, msg string) sesTypes.Message {
 }
 
 func (a *Alert) logLastError(lastError string, badRecipients []string) {
```

Comment:
Since you added more log messages, perhaps the name of this function should change too, maybe `logSendEmailError` or somesuch.

Reply:
It's still only used for logging the _last_ error, though the nothing about the function itself is limited to that.